code-reviews/example1/styles.css

```diff
@@ -0,0 +1,15 @@
+.main {
+    background-color: lightblue;
```

Comment:
should you add default styling using * selector like the box-sizzing, margin and padding before the main selector?